src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+
+	"override_enable_missing_command": "{emote_warn} Fehlender Befehl",
+	"override_enable_invalid_command": "{emote_warn} `{command}` ist kein gültiger befehl",
+	"override_enable_success": "{emote_success} ich habe erfolgreich den folgenden befehl aktiviert `{command}`",
+
+	"override_edit_invalid_part": "{emote_warn} Fehlendes argument `part`, es kann entweder `role` oder `command`",
+	"override_edit_missing_name": "{emote_warn} Fehlendes Argument `role/command name`",
+	"override_edit_missing_clearance": "{emote_warn} Fehlendes Argument `clearance`",
+	"override_edit_non_number": "{emote_warn} `{clearance}` ist kein Gültiger Typ",
+	"override_edit_less_than_0": "{emote_warn} die slowmode geschwindigkeit kann nicht kleiner oder gleich 0 sein",
+	"override_edit_more_than_100": "{emote_warn} die slowmode geschiwndigkeit kann nicht größer als 100 sein",
+	"override_edit_higher_than_yourself": "{emote_warn} Du kannst keinen slowmode erstellen der eine größere Geschwindigkeit hat",
+	"override_edit_invalid_role": "{emote_warn} Keine Gültige Rolle",
+	"override_edit_invalid_command": "{emote_warn} `{command}` ist kein Gültiger Befehl",
+	"override_edit_non_existent_override_command": "{emote_warn} Ich konnte keien Überschreibung finden, du musst erster eine Erstellen damit ich sie bearbeiten kann",
+	"override_edit_non_existent_override_role": "{emote_warn} Für die Rolle wurde keine Überschreibung gefunden",
+	"override_edit_success": "{emote_success} Erfolgreich die Überschreibung auf `{clearance}` geändert",
+
+	"override_remove_invalid_part": "{emote_warn} Fehlendes Argument `part`, Es kann entweder `role` oder `command`",
+	"override_remove_missing_name": "{emote_warn} Fehlendes Argument `role/command name`",
+	"override_remove_non_existent_override_command": "{emote_warn} Keine Befehl Überschreibung mit dem Namen gefunden",
+	"override_remove_non_existent_override_role": "{emote_warn} Keine Rollen überschreibung würde mit der id/mention gefunden",
+	"override_remove_success": "{emote_success} der befehl wurde erfolreich entfernt",
+
+	"automod_missing_limit": "{emote_warn} Fehlendes Argument `limit`, kann jeder Wert über `0` sein",
+	"automod_missing_item_remove": "{emote_warn} Fehlendes Argument `item` sollte vom Automod entfernt werden",
+	"automod_missing_punishment": "{emote_warn} Fehlendes Argument `punishment`, Es kann entweder `LOG`, `WARN`, `KICK` oder `BAN`",
+
+	"automod_non_number": "{emote_warn} `{limit}` ist keine gültige Nummer",
+	"automod_less_than_zero": "{emote_warn} `{limit}` kann nicht kleiner als 0 sein",
+
+	"automod_not_already_in_database": "{emote_warn} Item `{item}` ist noch nicht in der automod Datenbank",
+	"automod_already_in_database": "{emote_warn} Item `{item}` ist bereits in der automod datenbank",
+	"automod_added_to_the_database": "{emote_success}  `{item}` wurde zu `{part}` in den automod hinzugefügt",
```

Automod, real proofread tbd.
```suggestion
	"automod_added_to_the_database": "{emote_success} `{item}` wurde zu `{part}` in den automod hinzugefügt",
```